docs/sources/menu.yaml

```diff
@@ -29,171 +29,179 @@
+  - link: /features/panels/table_panel/
+    name: Table
+  - link: /features/panels/stat/
+    name: Stat
+  - link: /features/panels/gauge/
+    name: Gauge
+  - link: /features/panels/bar_gauge/
+    name: Bar gauge
+  - link: /features/panels/logs/
+    name: Logs
+  - link: /features/panels/singlestat/
+    name: Singlestat
+  - link: /features/panels/heatmap/
+    name: Heatmap
+  - link: /features/panels/alertlist/
+    name: Alert list
+  - link: /features/panels/dashlist/
+    name: Dashboard list
+  - link: /features/panels/text/
+    name: Text
+- name: Dashboard features
+  link: /features/dashboard/
+  children:
+  - link: /features/dashboard/dashboards/
+    name: Overview
+  - link: /reference/annotations/
+    name: Annotations
+  - link: /reference/dashboard_folders/
+    name: Folders
+  - link: /reference/playlist/
+    name: Playlist
+  - link: /reference/search/
+    name: Search
+  - link: /reference/share_dashboard/
+    name: Sharing a dashboard
+  - link: /reference/share_panel/
+    name: Sharing a panel
+  - link: /reference/timerange/
+    name: Time range
+  - link: /reference/export_import/
+    name: Export and import
+  - link: /reference/datalinks/
+    name: Data links
+  - link: /reference/dashboard_history/
+    name: Dashboard version history
+  - name: Keyboard Shortcuts
+    link: /features/shortcuts/
+  - name: Reporting
+    link: /features/reporting/
+  - link: /reference/dashboard/
+    name: JSON Model
+  - link: /reference/scripting/
+    name: Scripted dashboards
+- name: Data sources
+  link: /features/datasources/
+  children:
+  - link: /features/datasources/add-a-data-source/
+    name: Add data source
+  - link: /features/datasources/cloudwatch/
+    name: AWS Cloudwatch
+  - link: /features/datasources/azuremonitor/
+    name: Azure Monitor
+  - link: /features/datasources/elasticsearch/
+    name: Elasticsearch
+  - link: /features/datasources/stackdriver/
+    name: Google Stackdriver
+  - link: /features/datasources/graphite/
+    name: Graphite
+  - link: /features/datasources/influxdb/
+    name: InfluxDB
+  - link: /features/datasources/loki/
+    name: Loki
+  - link: /features/datasources/mssql/
+    name: Microsoft SQL Server
+  - link: /features/datasources/mixed/
+    name: MixedData
+  - link: /features/datasources/mysql/
+    name: MySQL
+  - link: /features/datasources/opentsdb/
+    name: OpenTSDB
+  - link: /features/datasources/postgres/
+    name: PostgreSQL
+  - link: /features/datasources/prometheus/
+    name: Prometheus
+  - link: /features/datasources/testdata/
+    name: TestData DB
+- name: Explore
+  link: /features/explore/
+- name: Alerting
+  link: /alerting/
+  children:
+  - link: /alerting/rules/
+    name: Engine and Rules
+  - link: /alerting/metrics/
+    name: Metrics
+  - link: /alerting/notifications/
+    name: Notifications
+- name: Image rendering
+  link: /administration/image_rendering/
+- name: Navigation links
+  link: /features/navigation-links/
```

Comment:
damit, this menu file is causing merge issue, I had already moved this item, it  should be under dashboard, not in root